ecs-agent/tmds/handlers/fault/v1/handlers/handlers.go

```diff
@@ -42,20 +46,31 @@ const (
 	faultInjectionEnabledError  = "fault injection is not enabled for task: %s"
 )
 
+var (
+	nsenterCommandString          = "nsenter --net=%s "
+	tcCheckInjectionCommandString = "tc -j q show dev %s parent 1:1"
+	tcAddQdiscRootCommandString   = "tc qdisc add dev %s root handle 1: prio priomap 2 2 2 2 2 2 2 2 2 2 2 2 2 2 2 2"
+	tcAddQdiscLossCommandString   = "tc qdisc add dev %s parent 1:1 handle 10: netem loss %d%%"
```

Comment:
Is it `%d%%` or `%d%`?

Reply:
It's `%d%%`. https://programming.guide/go/fmt-printf-percent-sign.html

Reply:
TIL